pkg/cmd/server/origin/controller.go

```diff
@@ -115,8 +118,36 @@ func (c *MasterConfig) NewOpenshiftControllerInitializers() (map[string]controll
 	deploymentTrigger := controller.DeploymentTriggerControllerConfig{Codec: codec}
 	ret["deploymenttrigger"] = deploymentTrigger.RunController
 
+	// initialize other controllers
+	imageTrigger := controller.ImageTriggerControllerConfig{
+		HasBuilderEnabled: c.Options.DisabledFeatures.Has(configapi.FeatureBuilder),
+		// TODO: make these consts in configapi
+		HasDeploymentsEnabled:  c.Options.DisabledFeatures.Has("triggers.image.openshift.io/deployments"),
+		HasDaemonSetsEnabled:   c.Options.DisabledFeatures.Has("triggers.image.openshift.io/daemonsets"),
+		HasStatefulSetsEnabled: c.Options.DisabledFeatures.Has("triggers.image.openshift.io/statefulsets"),
+		HasCronJobsEnabled:     c.Options.DisabledFeatures.Has("triggers.image.openshift.io/cronjobs"),
+	}
+	ret["image-trigger"] = imageTrigger.RunController
+
+	imageImport := controller.ImageImportControllerOptions{
+		MaxScheduledImageImportsPerMinute: c.Options.ImagePolicyConfig.MaxScheduledImageImportsPerMinute,
+		ResyncPeriod:                      10 * time.Minute,
+
+		DisableScheduledImport:                     c.Options.ImagePolicyConfig.DisableScheduledImport,
+		ScheduledImageImportMinimumIntervalSeconds: c.Options.ImagePolicyConfig.ScheduledImageImportMinimumIntervalSeconds,
+	}
+	ret["image-import"] = imageImport.RunController
+
 	templateInstance := controller.TemplateInstanceControllerConfig{}
 	ret["templateinstance"] = templateInstance.RunController
 
+	if c.Options.ControllerConfig.ServiceServingCert.Signer != nil {
+		serviceServingCert := controller.ServiceServingCertsControllerOptions{
```

Comment:
I think I prefer an unconditional add here with the skip happening in the init function.  That way the list of these things remains consistent.